src/routes/+page.svelte

```
@@ -101,6 +102,9 @@
 			case "Nil":
 				return NilEditor
 
+			case "QuickStart":
+				return QuickStartEditor
```

ESLint

[error] 'QuickStartEditor' is not defined.

(no-undef)